scm-ui/ui-webapp/src/repos/compare/CompareSelectorList.tsx

```diff
@@ -0,0 +1,279 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020-present Cloudogu GmbH and Contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import React, { FC, KeyboardEvent, useState } from "react";
+import { useTranslation } from "react-i18next";
+import classNames from "classnames";
+import styled from "styled-components";
+import { useBranches, useTags } from "@scm-manager/ui-api";
+import { Branch, Repository, Tag } from "@scm-manager/ui-types";
+import { Button, ErrorNotification, Loading, NoStyleButton, Notification } from "@scm-manager/ui-components";
+import DefaultBranchTag from "../branches/components/DefaultBranchTag";
+import CompareSelectorListEntry from "./CompareSelectorListEntry";
+import { CompareFunction, CompareProps, CompareTypes } from "./CompareSelectBar";
+
+type Props = {
+  onSelect: CompareFunction;
+  selected: CompareProps;
+  repository: Repository;
+  filter: string;
+};
+
+const TabStyleButton = styled(NoStyleButton)`
+  align-items: center;
+  border-bottom: var(--scm-border);
+  color: var(--scm-secondary-text);
+  display: flex;
+  justify-content: center;
+  margin-bottom: -1px;
+  padding: 0.5em 1em;
+  vertical-align: top;
+
+  &:hover {
+    border-bottom-color: var(--scm-hover-color);
+    color: var(--scm-hover-color);
+  }
+
+  &.is-active {
+    border-bottom-color: #33b2e8;
+    color: #33b2e8;
```

Comment:
No way to use a css class? With such hardcoded colors we are unable to introduce other themes, with a different color pallet.